.travis.yml

```diff
@@ -14,8 +14,20 @@ jobs:
         - pip3 install aura.tar.gz
       script:
         - python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.11 --no-upstream-fetch && python3 makeBuild.py
+    - # stage name not required, will continue to use `build`
+      name: "Run Vale against PR files"
```

Comment:
**issue:** Please also move this to a separate stage. I guess we should only run the vale check if Travis CI build passes. PS https://github.com/openshift/openshift-docs/pull/46864/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R41 to create a stage.

Reply:
Yes, I had it configured as such when I first did this. It seems logical to put the Vale error checking at the start of the build, however, this adds to the overall time for the build. 

I figured doing it like this was the most efficient way of doing the check.

Reply:
I still think this should be part of another stage, that way we only get the errors if travis build passes. If we include this in the `build` stage, it will generate more errors. For example, if Travis check fails for a PR, it will add errors to spreadsheet. Later, author will fix issues and push more changes in the same PR, which will cause same errors pushed to the spreadsheet again.